floss/main.py

```diff
@@ -146,7 +147,9 @@ def make_parser():
                       help="do not filter deobfuscated strings (may result in many false positive strings)",
                       action="store_true")
     parser.add_option("--max-instruction-count", dest="max_instruction_count", type=int, default=20000,
-                      help="maximum number of instructions to emulate per function")
+                      help="maximum number of instructions to emulate per function (default is 20000)")
+    parser.add_option("--max-address-revisits", dest="max_address_revisits", type=int, default=0,
```

Comment:
What's the reason this defaults to 0 and you later add 1 to it? As opposed to making the default value 1 and using that?

Reply:
my interpretation was that viv takes `max_hits` while the cli option is `max_revisits`, where the latter is `max_hits - 1`. keeping the terms the same would be more consistent, though i'm not sure which is easier to explain to a user on the cli.

Reply:
I really struggled to find a sane name.